packages/mongo-nav/src/mongo-select/README.md

```diff
@@ -0,0 +1,60 @@
+# Project Select
```

Comment:
Should we add information about OrgSelect too, if we're putting this here?

Reply:
I omitted this because we are not exporting OrgSelect, and only using it internally.